src/api.js

```diff
@@ -18,3 +18,18 @@ export const fetchArticleComments = async (article_id) => {
   const response = await kangorNews.get(`/articles/${article_id}/comments`);
   return response.data;
 };
+
+export const addVoteArticle = async (article_id, vote) => {
+  const newArticle = {
+    inc_votes: vote,
+  };
+  const response = await kangorNews.patch(`articles/${article_id}`, newArticle);
+  return response.data;
+};
+export const takeVoteArticle = async (article_id, vote) => {
+  const newArticle = {
+    inc_votes: vote - 2,
+  };
+  const response = await kangorNews.patch(`articles/${article_id}`, newArticle);
+  return response.data;
+};
```

Comment:
Nice that these api functions are organised into their own file. You could refactor these two functions to one function that changes the vote, but is passed a different amount to decrement when it's invoked